buildSrc/src/main/kotlin/plugin/AndroidLibraryPlugin.kt

```diff
@@ -25,20 +25,15 @@ class AndroidLibraryPlugin : BasePlugin() {
         get() = {
             implementation(
                 Library.daggerHiltAndroid,
-                Library.coroutines,
-                Library.room
+                Library.coroutines
             )
             testImplementation(
                 Library.junit,
                 Library.truth,
                 Library.mockito,
-                Library.androidXTest,
                 Library.coroutinesTest
             )
-            kapt(
-                Library.daggerHiltCompiler,
-                Library.roomCompiler
```

Comment:
Try not to add module specific dependencies to the general Android Library plugin

Reply:
Okay, thank you